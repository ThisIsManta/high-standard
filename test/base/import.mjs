import 'c'
import a from './a'
import b from 'b'
import React from 'react'

void a
void b
void React
