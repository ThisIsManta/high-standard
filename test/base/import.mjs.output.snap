import React from 'react'
import b from 'b'

import a from './a'

import 'c'

void a
void b
void React
